spec/models/user_spec.rb

```diff
@@ -33,14 +33,14 @@
       @order_3 = create(:cancelled_order)
       @order_4 = create(:shipped_order)
 
-      @item_1 = create(:item, user: @user)
-      @item_2 = create(:item, user: @user)
-      @item_3 = create(:item, user: @user)
-      @item_4 = create(:item, user: @user)
-      @item_5 = create(:item, user: @user)
-      @item_6 = create(:item, user: @user)
-      @item_7 = create(:item, user: @user)
-      @item_8 = create(:item, user: @user)
+      @item_1 = create(:item, user: @merchant)
```

Comment:
I changed this from user to merchant as only merchants have items. I changed the tests below it also, so I am now not confused when reading it 😄